test-fixtures/postgres/default_values.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE default_values (
+    id SERIAL,
+    bool_null BOOLEAN DEFAULT NULL,
+    bool_cst BOOLEAN DEFAULT TRUE,
+    int_cst INTEGER DEFAULT 42,
+    str_null VARCHAR(25) DEFAULT NULL,
+    str_cst VARCHAR(25) DEFAULT 'constant',
+    str_expr VARCHAR(25) DEFAULT UPPER('Hello' || 'World'),
+    date_null TIMESTAMP DEFAULT NULL,
+    date_cst1 TIMESTAMP DEFAULT '2010-01-01T00:00:00Z',
+    date_cst2 TIMESTAMP DEFAULT '1983-05-27',
+    date_expr1 TIMESTAMP DEFAULT NOW(),
+    date_expr2 TIMESTAMP DEFAULT now(),
```

Comment:
I don't see your fix 🤔 . You were supposed to fix CURRENT_TIMESTAMPS so it should be in the tests 😉

Reply:
I did that initially, but it is a bit of a pain as the output of the test is different depending the postgres version.
I think that on Postgres 9.x, CURRENT_TIMESTAMP() was just an alias to NOW(), and it is no longer the case in 12.x (maybe it was the opposite)

If you do `CREATE TABLE ... somecolumn TIMESTAMP DEFAULT CURRENT_TIMESTAMP`
And then later on:
`SELECT column_default FROM information_schema.columns WHERE .... AND column_name = 'somecolumn'`

On Postgres 9.x you get: `NOW()`
On Postgres 12.x you get: `CURRENT_TIMESTAMP`

This breaks the tests, as the same output is expected for all databases and versions, which is not feasible here.
I would need to provide 6 different expected outputs in `test-expected` for each one of the database backends which are tested, which was asking for more refactoring, and the PR was long enough